index.html

```diff
@@ -19,8 +19,78 @@
     <div class="container">
       <div class="row">
           <div class="col-sm-12">
-              <h1>Start Here!</h1>
+            <div class="jumbotron text-center">
+            <h1 class="display-4">Julian Alexander Piñeyro</h1>
+            
+            <img src="https://goo.gl/UQp1kg" class="img-fluid rounded-circle" alt="photo" width="33%" height="100%">
```

Comment:
Another thing is that for images like this, I'd probably use a local image rather than a goo.gl link. I think local would load faster and be more performant, and in general shortened URLs can behave a little bit funky. Again, not the biggest deal because it's obviously working as is, but worth considering.